include/seqan3/alignment/matrix/alignment_optimum.hpp

```diff
@@ -17,7 +17,7 @@
 #include <seqan3/alignment/matrix/detail/matrix_coordinate.hpp>
 #include <seqan3/core/concept/core_language.hpp>
 #include <seqan3/core/type_traits/template_inspection.hpp>
-#include <seqan3/core/simd/concept.hpp>
+#include <seqan3/utility/simd/concept.hpp>
```

Comment:
The header is now at the wrong position. Even if we do other header rearrangements in separate commits, we should place the new header directly into the correct header-order-location.